pokemongo_bot/__init__.py

```diff
@@ -712,13 +712,123 @@ def _setup_api(self):
         self.login()
         # chain subrequests (methods) into one RPC call
 
+        self._check_tutorial_state()
         self._print_character_info()
         self.api.activate_signature(self.get_encryption_lib())
         self.logger.info('')
         self.update_inventory()
         # send empty map_cells and then our position
         self.update_web_location()
 
+    def _check_tutorial_state(self):
```

Comment:
This should definitely not live in `__init__`. Can you move this to another file / class with this code and call it from here.